docs/release.rst

```diff
@@ -293,24 +293,10 @@ First checkout the ``f<version>-release`` branch and merge ``f<version>-devel``
     git merge --no-ff f<version>-devel
 
 
-Then correct pykickstart version for the new Fedora release by changing all occurrences of:
-
-::
-
-    returnClassForVersion()
-
-to
-
-::
-
-    returnClassForVersion(version=F<new version>)
-
-Commit the result. The commit will become one of the few exclusive release branch commits,
-as we can't let it be merged back to master via the devel branch for obvious reasons.
-
```

Comment:
It is still necessary to update the pykickstart version in these files:

https://github.com/rhinstaller/anaconda/blob/master/pyanaconda/core/kickstart/version.py#L21
https://github.com/rhinstaller/anaconda/blob/master/dracut/parse-kickstart#L46

Reply:
So DEVEL should be replaced by F<version> in these files ? I though that that was not needed and the F29 pykickstart package will just continue to point DEVEL to F29, while rawhide pykickstart builds will switch to DEVEL == F30.

Reply:
Yes. I don't think that pykickstart does that. It doesn't have a branch for F29.

Reply:
Ah, OK. So I'll adjust the docs & set the kickstart version on f29-release branch.